packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -144,7 +145,18 @@ module.exports = {
     // https://github.com/facebook/create-react-app/issues/290
     // `web` extension prefixes have been added for better support
     // for React Native Web.
-    extensions: ['.mjs', '.web.js', '.js', '.json', '.web.jsx', '.jsx'],
+    extensions: [
+      '.mjs',
+      '.web.js',
+      '.js',
+      '.json',
+      '.web.jsx',
+      '.jsx',
+      '.web.ts',
+      '.ts',
```

Comment:
`.ts` should probably come before `json`, and let's not add the `web` variants. This was for node packages only.

Reply:
@Timer It's needed for react-native-web projects which cra supports

Reply:
IIRC this was only for imported packages, and TS shouldn't be shipped to npm.

Reply:
Had to keep it otherwise importing typescript files would not work